StreamChat.xcodeproj/project.pbxproj

```diff
@@ -1004,6 +1004,13 @@
 			remoteGlobalIDString = 799C941A247D2F80001F1104;
 			remoteInfo = StreamChat;
 		};
+		ADE787132617A410007129ED /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 8AD5EC8622E9A3E8005CFAC9 /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = 793060E525778896005CF846;
+			remoteInfo = StreamChatTestTools;
+		};
```

Comment:
nit: would be nice to have this commit as the first one so the reviewer expects `import StreamChatTestTools` later in the PR

Reply:
Yes makes sense 👍 Although this deserves a PR on its own, I think it is not worth it adding new mocks to the StreamChat SDK, or new dummies, if we should be using the StreamChatTestTools. So in this case for it made sense to start using SteamChatTestTools for LLC too, so we can slowly start using it, and deprecating the old dummies/mocks etc..

Reply:
✅